manta.wdl

```diff
@@ -4,31 +4,30 @@ import "common.wdl"
 
 task Somatic {
     input {
-        IndexedBamFile tumorBam
-        IndexedBamFile? normalBam
-        Reference reference
-        String runDir
+        File tumorBam
+        File tumorBamIndex
+        File? normalBam
+        File? normalBamIndex
+        File referenceFasta
+        File referenceFastaFai
+        String runDir = "."
         File? callRegions
         File? callRegionsIndex
         Boolean exome = false
 
         Int cores = 1
         Int memory = 4
-        String dockerTag = "1.4.0--py27_1"
+        String dockerImage = "quay.io/biocontainers/manta:1.4.0--py27_1"
 
-        File? doNotDefineThis #FIXME
     }
 
-    File? normalBamFile = if defined(normalBam)
-        then select_first([normalBam]).file
-        else doNotDefineThis
-
     command {
+        mkdir -p ~{runDir}
```

Comment:
Manta will create the rundir itself, so this is not necessary. It might actually crash like this, complaining that the output directory already exists.